Waffle march 1

Thinking-waffle opened the pull request:
-Guiana shared tree if you form it.
-Tweaks to r56 TOA formables
-Fixed oversight in Mali and Nigeria

Thinking-waffle committed:
Add files via upload

Thinking-waffle committed:
Update generic.txt

added  guiana unification tree

Thinking-waffle committed:
Update r56_formables_l_english.yml

Thinking-waffle committed:
Update mali.txt

Thinking-waffle committed:
Update mali.txt

Thinking-waffle committed:
Update nigeria.txt

Thinking-waffle committed:
Added TOA formables file with r56 tweak

Thinking-waffle committed:
Update CAN_organization.txt

start of dlc integration (to be continued in this file)
fixes a bug too this time

Thinking-waffle committed:
Update Lemur_focus_events.txt

Don't send migrants on tiny islands, enclaves and wastelands.

Thinking-waffle committed:
Update korea.txt

Likely fixed a bug where it looked for an overlord but could be selected when it wasn't a puppet;

marijn211 reviewed (dismissed):
Ported decision file needs to have states corrected, I can do that later

Thinking-waffle committed:
Update land_doctrine.txt

ported new bonus of the people's  war from vanilla

Thinking-waffle committed:
fix stalin becoming marshal while not in power

Thinking-waffle committed:
Update LAR_NewsEvents.txt

Non communist russia shouldn't puppet victorious communist Spain if they expanded Soviet aid.

Thinking-waffle committed:
Small idel ural change

There are two decisions to form something close for Tatarstan, maybe that should be changed in the future.

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-march-1

marijn211 committed:
Fixed state references in ported formables

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-march-1

marijn211 committed:
Merge branch 'Beta-Edits' into waffle-march-1

marijn211 committed:
Error log thingies

marijn211 committed:
Broken check in Guyana branch
